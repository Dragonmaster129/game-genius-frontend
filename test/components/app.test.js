import React from "react";
import renderer from "react-test-renderer";
import App from "../../src/components/app";

test("app is the same as it was last time", () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
