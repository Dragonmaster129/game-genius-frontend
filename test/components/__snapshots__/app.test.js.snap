// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app is the same as it was last time 1`] = `
<div
  className="app hz"
>
  <div
    className="player"
  >
    <div
      className="v playerCard"
    >
      <div
        className="heading"
      >
        <h1>
          Player card
        </h1>
        <div
          className="hz"
        >
          <h2>
            Profession: 
            Doctor
          </h2>
          <h2>
            Player: 
            3
          </h2>
          <h2>
            Auditor: 
            Sam
          </h2>
        </div>
      </div>
      <hr />
      <h2>
        Income Statement
      </h2>
      <div
        className="income-wrapper"
      >
        <h2
          className="title"
        >
          Income
        </h2>
        <div
          className="hz"
        >
          <div
            className="v income-categories"
          >
            <h3>
              Salary: 
              13200
            </h3>
            <div
              className="interest category"
            >
              <h3
                className="category-click"
                onClick={[Function]}
              >
                Interest: 0
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-angle-down "
                  data-icon="angle-down"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 384 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M192 384c-8.188 0-16.38-3.125-22.62-9.375l-160-160c-12.5-12.5-12.5-32.75 0-45.25s32.75-12.5 45.25 0L192 306.8l137.4-137.4c12.5-12.5 32.75-12.5 45.25 0s12.5 32.75 0 45.25l-160 160C208.4 380.9 200.2 384 192 384z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
              </h3>
              <div
                className="item-wrapper hidden"
              />
            </div>
            <div
              className="dividends category"
            >
              <h3
                className="category-click"
                onClick={[Function]}
              >
                Dividends: 0
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-angle-down "
                  data-icon="angle-down"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 384 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M192 384c-8.188 0-16.38-3.125-22.62-9.375l-160-160c-12.5-12.5-12.5-32.75 0-45.25s32.75-12.5 45.25 0L192 306.8l137.4-137.4c12.5-12.5 32.75-12.5 45.25 0s12.5 32.75 0 45.25l-160 160C208.4 380.9 200.2 384 192 384z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
              </h3>
              <div
                className="item-wrapper hidden"
              />
            </div>
            <div
              className="real-estate category"
            >
              <h3
                className="category-click"
                onClick={[Function]}
              >
                Real Estate: 0
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-angle-down "
                  data-icon="angle-down"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 384 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M192 384c-8.188 0-16.38-3.125-22.62-9.375l-160-160c-12.5-12.5-12.5-32.75 0-45.25s32.75-12.5 45.25 0L192 306.8l137.4-137.4c12.5-12.5 32.75-12.5 45.25 0s12.5 32.75 0 45.25l-160 160C208.4 380.9 200.2 384 192 384z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
              </h3>
              <div
                className="item-wrapper hidden"
              />
            </div>
            <div
              className="businesses category"
            >
              <h3
                className="category-click"
                onClick={[Function]}
              >
                Businesses: 0
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-angle-down "
                  data-icon="angle-down"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 384 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M192 384c-8.188 0-16.38-3.125-22.62-9.375l-160-160c-12.5-12.5-12.5-32.75 0-45.25s32.75-12.5 45.25 0L192 306.8l137.4-137.4c12.5-12.5 32.75-12.5 45.25 0s12.5 32.75 0 45.25l-160 160C208.4 380.9 200.2 384 192 384z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
              </h3>
              <div
                className="item-wrapper hidden"
              />
            </div>
          </div>
          <div
            className="v income-main"
          >
            <h3>
              Passive Income: 
              0
            </h3>
            <h3>
              Total Income: 
              13200
            </h3>
          </div>
        </div>
      </div>
      <hr />
      <div
        className="expenses hz"
      >
        <div
          className="v"
        >
          <h2>
            Expenses
          </h2>
          <h3>
            Taxes: 
            3420
          </h3>
          <h3>
            Home Mortgage: 
            1900
          </h3>
          <h3>
            School Loan Payment: 
            750
          </h3>
          <h3>
            Car Payment: 
            380
          </h3>
          <h3>
            Credit Card Payment: 
            270
          </h3>
          <h3>
            Retail Payment: 
            50
          </h3>
          <h3>
            Other Expenses: 
            2880
          </h3>
          <h3>
            Child Expenses: 
            0
          </h3>
          <h3>
            Bank Loan Payment: 
            0
          </h3>
        </div>
        <div
          className="v"
        >
          <h3>
            Number of Children: 
            0
          </h3>
          <h3>
            Per Child Expense: 
            640
          </h3>
          <h3>
            Total Expenses: 
            9650
          </h3>
        </div>
      </div>
      <hr />
      <div
        className="v right"
      >
        <h3
          className="cash-flow"
        >
          Monthly Cash Flow: 
          3550
        </h3>
        <h3
          className="cash"
        >
          Cash: 
          3950
        </h3>
      </div>
      <hr />
      <div
        className="hz"
      >
        <div
          className="v"
        >
          <h2>
            Assets
          </h2>
          <h3>
            Savings
          </h3>
          <div
            className="stock"
          >
            <div
              className="hz"
            >
              <h3>
                Stocks/Mutual's/CDs
              </h3>
              <h3>
                Number of Shares
              </h3>
              <h3>
                Cost/Share
              </h3>
            </div>
          </div>
          <div
            className="re"
          >
            <div
              className="hz heading"
            >
              <h3>
                Real Estate
              </h3>
              <h3>
                Down Pay
              </h3>
              <h3>
                Cost
              </h3>
            </div>
          </div>
          <div
            className="business"
          >
            <div
              className="hz"
            >
              <h3>
                Business
              </h3>
              <h3>
                Down Pay
              </h3>
              <h3>
                Cost
              </h3>
            </div>
          </div>
        </div>
        <hr />
        <div
          className="v"
        >
          <h2>
            Liabilities
          </h2>
          <h3>
            Home Mortgage: 
            202000
          </h3>
          <h3>
            School Loans: 
            150000
          </h3>
          <h3>
            Car Loans: 
            19000
          </h3>
          <h3>
            Credit Cards: 
            9000
          </h3>
          <h3>
            Retail Debt: 
            1000
          </h3>
          <h3>
            RE Mortgage: 
          </h3>
          <h3>
            Bank Loan: 
            0
          </h3>
        </div>
      </div>
    </div>
    <hr />
    <div
      className="actions hz"
    >
      <div
        className="payday"
      >
        <button
          onClick={[Function]}
        >
          Payday
        </button>
      </div>
      <div
        className="buy"
      >
        <button
          onClick={[Function]}
        >
          Buy
        </button>
      </div>
      <div
        className="doodad"
      >
        <button
          onClick={[Function]}
        >
          Doodad
        </button>
      </div>
      <div
        className="sell"
      >
        <button
          onClick={[Function]}
        >
          Sell
        </button>
      </div>
      <div
        className="pay-loan"
      >
        <button
          onClick={[Function]}
        >
          Pay Loan
        </button>
      </div>
    </div>
  </div>
  <div
    className="card"
  >
    
    
    <div
      className="loan"
    />
  </div>
  <div
    className="spacer"
  />
</div>
`;
